src/Commands/LowGoalShoot.cpp

```diff
@@ -0,0 +1,11 @@
+#include "LowGoalShoot.h"
+#include "SetArms.h"
+#include "SetLifter.h"
+#include "ShooterControl.h"
+
+LowGoalShoot::LowGoalShoot()
+{
+  AddSequential(new SetArms(Intake::kUp));
+  AddParallel(new SetLifter(Shooter::kDown));
```

Comment:
Same issue as before, should we include Subsystem even thought commands included already include them